Gemfile

```diff
@@ -2,3 +2,5 @@ source 'https://rubygems.org'
 
 # Specify your gem's dependencies in twofishes.gemspec
 gemspec
+
+gem 'activesupport', :require => false
```

Comment:
i think this definition can be removed. after adding it to the gemspec, it still needs to be required manually.